docs/dev/marketplace_offering.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Build A Marketplace Offering
+type: documentation
+aliases: 
+- 'developers/marketplace/'
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"
+further_reading:
+- link: "https://www.datadoghq.com/partner/"
+  tag: "Partner Network"
+  text: "Datadog Partner Network"
+- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
+  tag: "Blog"
+  text: "Expand your monitoring reach with the Datadog Marketplace"
+- link: "/developers/integrations/marketplace_offering/"
+  tag: "Documentation"
+  text: "Learn how to develop a Marketplace offering"
+- link: "/developers/datadog_apps/"
+  tag: "Documentation"
+  text: "Learn about Datadog Apps"
+---
+
+## Overview
+
+The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their paid offerings to Datadog users.  
+
+While the **Integrations** page includes integrations built by both Datadog and Technology Partners at no cost, the **Marketplace**  is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, software licenses, and professional services.
+
+## List an offering 
+
+All Technology Partners can list an out-of-the-box integration on the **Integrations** page, or a commercial paid offering on the **Marketplace** page. The following types of offerings are supported on the Datadog Marketplace:
+
+### Integrations
+Marketplace integrations that submit third-party data or pull data from a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data types, such as metrics, events, logs, traces, and more.
+
+### Software licenses
+Software licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.
+
+### Professional services
+[Professional services][18] enable you to offer your team's services for implementation, support, or management for a set period of time.
+
+## Why join the Datadog Marketplace? 
+
+Marketplace Partners have unique benefits that are not available to partners who list out-of-the-box integrations:
+ 
+  - Go-to-Market collaboration, including a blog post, a quote for a press release, and social media amplification, with access to dedicated sales and marketing resources focused on accelerating partner growth. 
+
+  - Training and Support for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.
+
+  - Exclusive opportunities to sponsor conferences and events (such as Datadog DASH) at a discounted rate.
+  
+  - Generating new leads from user discovery
+
+{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}
+
+## Join the Datadog partner network
```

Comment:
Should this be moved up?

Reply:
I think it makes sense here - the flow is that a user will read about what the marketplace is and what they can list there, understand the benefits of the marketplace, and then apply to be a tech partner if all of that sounds intriguing.